data/ecosystem/vendors.yaml

```diff
@@ -41,6 +41,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: Cribl
```

Comment:
@nheudecker please add Cribl in line 1 to the list of words that should be ignored by the spell checker

Reply:
Looks like I somehow added Cribl successfully.